.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           - {os: ubuntu-16.04, ruby: 2.4, db: mariadb10.0, allow-failure: true}
           # Comment out due to ci/setup.sh stucking.
           # - {os: ubuntu-18.04, ruby: 2.4, db: mariadb10.1}
-          # `service mysql restart` fails.
+          # Allow failure due to the issue #965, #1165.
```

Comment:
I opened a PR #1193 to fix #965 and #1165 . After the PR will be merged, possibly the MariaDB 10.3 case passes.
And we can remove the comment line and `allow-failure: true` too.